scripts/after_install.sh

```diff
@@ -45,14 +45,15 @@ psql -U postgres -d just-spaces -c "CREATE EXTENSION IF NOT EXISTS postgis"
 
 # OPTIONAL Run migrations and other management commands that should be run with
 # every deployment
-$VENV_DIR/bin/python $PROJECT_DIR/manage.py migrate
-$VENV_DIR/bin/python $PROJECT_DIR/manage.py createcachetable
-$VENV_DIR/bin/python $PROJECT_DIR/manage.py collectstatic --no-input
+sudo -H -u datamade $VENV_DIR/bin/python $PROJECT_DIR/manage.py migrate
```

Comment:
This was needed to run migrations but isn't reflected in current SOP here: https://github.com/datamade/deploy-a-site/blob/master/templates/django/%7B%7Bcookiecutter.directory_name%7D%7D/scripts/after_install.sh#L48

I'm honestly not sure why this ever works. Do you think this is worth opening an issue in `deploy-a-site`?

Reply:
Yeah, this is pretty weird behavior. Do you mind opening an issue in `devops` to track this?

Reply:
Opened! https://github.com/datamade/devops/issues/105